examples/raytracing/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_executable(ArborX_RayTracing.exe example_raytracing.cpp)
+target_link_libraries(ArborX_RayTracing.exe ArborX::ArborX Boost::program_options)
+add_test(NAME ArborX_RayTracing_Example COMMAND ./ArborX_RayTracing.exe --spheres=100 --rays=10000 --L=100.0)
```

Comment:
Why are we passing command line arguments if we just specifies the default values?